README.md

```diff
@@ -19,16 +18,92 @@
 * Can drag a View Controller onto the canvas.
 * Can locate and drag items from the Object Library.
 * Can locate the Attributes Inspector and change the Background color of Views
-* They know how to connect the ViewController.swift file and understand it's view life cycle.
+* They know how to connect the `ViewController.swift` file and understand its view life cycle.
 * Can hook up IBOutlets and IBActions.
 * Understands UIKit.
 
-## Outline / Notes
+## Paints: A Primer
+
+Have you ever mixed two colors of paint together? You get a different color, right? In this lab, you will be mixing two colors of paint together to produce another color.
+
+Artists often use _primary_ colors to produce _secondary_ colors. Primary colors are ones that can easily be mixed together to produce other colors. For example, you can mix red and blue together to make purple, or yellow and red together to make orange. Here is a table showing a primary color along the top and left side, and what colors they produce when mixed:
+
+<table>
+<tr>
+<th></th>
+<th>Red</th>
+<th>Yellow</th>
+<th>Blue</th>
+</tr>
+<tr>
+<th>Red</td>
+<td>Red</td>
+<td>Orange</td>
+<td>Purple</td>
+</tr>
+<tr>
+<th>Yellow</th>
+<td>Orange</td>
+<td>Yellow</td>
+<td>Green</td>
+</tr>
+<tr>
+<th>Blue</th>
+<td>Purple</td>
+<td>Green</td>
+<td>Blue</td>
+</tr>
+</table>
+
+In this lab, you will be implementing this logic to update an app UI to display these different colors.
+
+## App
+
+Specifically, you are tasked with producing a user interface for an app that looks like this:
+
+![Paint Bucket UI](http://i.imgur.com/kIaCdAy.png)
+
+The user can select different options in order to show different colors in the paint bucket. For example, selecting **Blue** and **Yellow** would produce this:
+
+![Paint Bucket UI](http://i.imgur.com/8mHzYjT.png)
+
+Selecting **Blue** and **Red** would show this:
+
+![Paint Bucket UI](http://i.imgur.com/Se2uNIl.png)
+
+You get the idea by now.
+
+As in previous labs, some parts of this app have already been written for you, but you will need to fill in the missing parts in order for the application to work. Open up the Xcode project provided by in this repo from `UIFun/UIFun.xcodeproj` and check out what has been given to you.
+
+### Starting Off
+
+The project contains a barebones (and barely functional) view controller, in `ViewController.swift`. You'll see that several parts have been marked as `TODO`. This indicates the parts of the code that _you_ have to fill in yourself.
+
+`Main.storyboard` contains a single scene. That scene already has a view, the paint bucket. The segmented controls have not been created for you yet. No IB outlets have been set up for you, either, and no UI elements have been connected to the IB action declared in `ViewController.swift`, either.
```

Comment:
I think this needs to read..

The segmented controls have been created for you.

It seems in your instructions below that you don't include a part to create them in interface builder, Task 2 & 3 have them connecting outlets to segmented controls. I'm OK with this, I would be more interested in having them create the segmented controls themselves first (which might have been your intention here).

Let me know how you want to proceed, I'm all good either way.. but I'm leaning more towards having them create them on their own.